DQMOffline/Trigger/plugins/METplusTrackMonitor.cc

```diff
@@ -0,0 +1,461 @@
+#include "DQMOffline/Trigger/plugins/METplusTrackMonitor.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "DQM/TrackingMonitor/interface/GetLumi.h"
+
+#include "CommonTools/TriggerUtils/interface/GenericTriggerEventFlag.h"
+
+#include "DataFormats/Math/interface/deltaPhi.h"
+#include "DataFormats/Math/interface/deltaR.h"
+
+// -----------------------------
+//  constructors and destructor
+// -----------------------------
+
+METplusTrackMonitor::METplusTrackMonitor( const edm::ParameterSet& iConfig ) :
+  folderName_             ( iConfig.getParameter<std::string>("FolderName") )
+  , metToken_             ( consumes<reco::CaloMETCollection>      (iConfig.getParameter<edm::InputTag>("met")            ) )
+  , hltMetToken_          ( consumes<reco::CaloMETCollection>      (iConfig.getParameter<edm::InputTag>("onlineMet")      ) )
+  , hltMetCleanToken_     ( consumes<reco::CaloMETCollection>      (iConfig.getParameter<edm::InputTag>("onlineMetClean") ) )
+  , muonToken_            ( consumes<reco::MuonCollection>         (iConfig.getParameter<edm::InputTag>("muons")          ) )
+  , jetToken_             ( consumes<reco::PFJetCollection>        (iConfig.getParameter<edm::InputTag>("jets")           ) )
+  , vtxToken_             ( consumes<reco::VertexCollection>       (iConfig.getParameter<edm::InputTag>("vertices")       ) )
+  , theTrigSummary_       ( consumes<trigger::TriggerEvent>        (iConfig.getParameter<edm::InputTag>("trigSummary")    ) )
+  , hltTrk50FilterTag_       ( iConfig.getParameter<edm::InputTag>("isoTrkFilter") )
+  , met_variable_binning_    ( iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<std::vector<double> >("metBinning") )
+  , muonPt_variable_binning_ ( iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<std::vector<double> >("ptBinning")  )
+  , met_binning_          ( getHistoPSet   (iConfig.getParameter<edm::ParameterSet>("histoPSet").getParameter<edm::ParameterSet>("metPSet")    ) )
```

Comment:
Which of these are you referring to? If you mean this variable binning histograms, we don't particularly need them; it seemed to be a common inclusion from other modules.